src/Features/VisualBasic/Portable/UseCompoundAssignment/VisualBasicUseCompoundAssignmentDiagnosticAnalyzer.vb

```diff
@@ -20,5 +20,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseCompoundAssignment
         Protected Overrides Function GetAnalysisKind() As SyntaxKind
             Return SyntaxKind.SimpleAssignmentStatement
         End Function
+
+        Protected Overrides Function IsSupported(assignmentKind As SyntaxKind, options As ParseOptions) As Boolean
+            Return True
```

Comment:
True? This isn't a feature in VB, please add a test.

Reply:
This is effectively saying: "all assignments are supported in VB regardless of hte version".  It is effectively tested with all the existing VB tests for this feature :)

Reply:
Ah, I understand.